schemas/protocol.fbs

```diff
@@ -86,6 +86,10 @@ table CheckpointBeacon {
     hash_prev_block: Hash (required);
 }
 
+table GetBlocks {
+    highest_block_checkpoint: CheckpointBeacon;
```

Comment:
To avoid having unnecessary `Options` around our code, i would advise to use the `required` attribute. 
```suggestion
    highest_block_checkpoint: CheckpointBeacon (required);
```